lighthouse-core/gather/gather-runner.js

```diff
@@ -336,9 +348,16 @@ class GatherRunner {
           // To reach this point, all errors are non-fatal, so return err to
           // runner to handle turning it into an error audit.
           artifacts[gathererName] = err;
+          // Track page load errors separately, so we can fail loudly if needed.
+          if (err.code === 'PAGE_LOAD_ERROR') pageLoadFailures.push(err);
         });
       });
     }, Promise.resolve()).then(_ => {
+      // Fail the run if more than 50% of all audits failed due to page load failure.
```

Comment:
"50% of all audits" => "50% of all artifacts"

Reply:
oh oops, yeah done :)